.travis.yml

```diff
@@ -91,8 +93,10 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test
+            - llvm-toolchain-trusty-3.9
           packages:
             - g++-6
+            - llvm-3.9-dev=1:3.9*
```

Comment:
We could loosen this to just `llvm-3.9-dev`, or pin this to `llvm-3.9-dev=1:3.9~svn288847-1~exp1`, but I'm not sure we should.